src/redux-middleware/updateEditingValue.ts

```diff
@@ -13,7 +13,7 @@ const updateEditingValue: ThunkMiddleware<State> = ({ getState }) => {
     const thought = state.cursor ? getThoughtById(state, head(state.cursor)) : null
     const value = thought?.value ?? null
 
-    editingValueStore.update(value)
+    editingValueStore.update(value?.trim() ?? null)
```

Comment:
Could you please give a quick explanation for this change? It seems like it might have been related to previous attempts to resolve the whitespace handling for divider widths, but I don't see how it affects the current solution.